sensorhub-driver-android/src/main/java/org/sensorhub/impl/sensor/android/video/AndroidCameraOutput.java

```diff
@@ -209,16 +194,13 @@ public void run()
     protected void initVideoCapture(Camera.CameraInfo info) throws SensorException {
         // default for Most of the CODECs
         // if camera was successfully opened, prepare for video capture
-        if (camera != null)
-        {
-            try
-            {
+        if (camera != null) {
+            try {
                 Parameters camParams = camera.getParameters();
 
                 // set video capture and encodign options
                 VideoEncoderConfig videoConfig = parentSensor.getConfiguration().videoConfig;
                 frameRate = videoConfig.frameRate;
-                selectedPreset = videoConfig.selectedPreset;
```

Comment:
Sorry I didn't look at this file before.
Why was this line removed?

Reply:
Should not have been. I was trying something there and it got caught up in taking that code back out. The fact that selected preset was defined elsewhere happened to keep me and the compiler from catching it.

Pushed a commit with the line added back.